### PR TITLE
restructure and break content into concise sections

### PR DESCRIPTION
**Pull request by cvan**
(Part of this is based off of my other PR, #6.) I've heard a lot of confusion from folks in regards to what this workshop is about and how they attend. I adjusted the content to include the essentials and rearranged the content into sections with FAQ-style headings. I think it reads quite well, but I'd love to hear people's thoughts.

I pushed a copy of the changes to my fork's GitHub Pages so folks can preview it:
https://cvan.io/vr-workshop/
https://cvan.io/vr-workshop/sponsorship.html

And, for measure:
https://w3c.github.io/vr-workshop/
https://w3c.github.io/vr-workshop/sponsorship.html

### CODE REVIEW

**Comment by cvan** [comment]
I would event suggest removing the "How is the Web a viable platform for VR?" and "Why does the Web platform need VR?" sections. Not sure how much of this explanatory text belongs in a Workshop overview page.

And, as mentioned in issue #7 I filed, I think the text of the "position statement" section (the section now titled "How can I suggest a presentation?") can be shortened – at the very least, the bullet-pointed requirements. Is it necessary to say "preferably in HTML or plain-text format; images should be included inline in HTML using base64-encoded data URIs"? It seems a bit formal and IMO adds unnecessary text for people to read/scan.

**Review by dontcallmedom — APPROVED** [review]
These changes work for me, but would like to get @anssiko views as well

File: index.html [hunk 1]
```diff
@@ -17,12 +17,11 @@ <h1 id="top">
       W3C Workshop on Web &amp; Virtual Reality
     </h1>
     <p>
-      October 2016, 19-20, San Jose, CA, USA
+      October 19-20, 2016; San Jose, CA, USA
```
Comment (cvan) [inline index.html:20]: it might be useful to include the venue below here so people don't have to dig through to the Logistics section. thoughts?

**Comment by cvan** [comment]
Made a few more adjustments ([see preview](https://cvan.io/vr-workshop/)):
- Moved the [`How is the Web a viable platform for VR?`](https://cvan.io/vr-workshop/#why-the-web) and [`Why does the Web platform need VR?`](https://cvan.io/vr-workshop/#why-vr) headings from `<h2>`s to `<h3>`s under the [`Goals` (`What is the purpose of this workshop?`)](https://cvan.io/vr-workshop/#goals) section.
- Renamed the [`Logistics` heading to `Location`](https://cvan.io/vr-workshop/#logistics) since that's the only item in that section. (I didn't want to change the anchor in case anyone was previously linking to it, but I'd be willing to change it to `location`.)
- Cleaned up the [`How can I attend?` section](https://cvan.io/vr-workshop/#participate).

**Comment by cvan** [comment]
Also these changes ([see preview](https://cvan.io/vr-workshop/)):
- Decided to change the `id` of [`Location`](https://cvan.io/vr-workshop/#location) from `logistics` to `location` to reflect the heading text.
- Tightened up the language for the [`Topics`](https://cvan.io/vr-workshop/#topics) section.
- Changed the dates in the [`Dates` sidebar](https://cvan.io/vr-workshop/#dates) from European dates to US dates, for the sake of consistency and readibility (targeting the majority of the expect audience).

**Comment by anssiko** [comment]
LGTM. Thanks for the PR! Feel free to merge at will.

**Pull request merged**